src/resources/extensions/quarto/docusaurus/_extension.yml

```
@@ -17,9 +17,13 @@ contributes:
     format: docusaurus-md
   formats:
     md:
+      # Although we use a custom writer, we still need the variants here the lua filters to render correctly.
+      # Ideally, we would forward the variants to the custom writer.
+      variant: gfm+pipe_tables+tex_math_dollars+header_attributes+raw_html+all_symbols_escapable+backtick_code_blocks+fenced_code_blocks+space_in_atx_header+intraword_underscores+lists_without_preceding_blankline+shortcut_reference_links
       writer: docusaurus_writer.lua
       output-ext: md
       inline-includes: true
+      preserve-yaml: true
```

Going with my other comment I believe adding the variant shoudl work 

```suggestion
      # Although we use a custom writer, we still need the variants here the lua filters to render correctly.
      # Ideally, we would forward the variants to the custom writer.
      variant: gfm+pipe_tables+tex_math_dollars+header_attributes+raw_html+all_symbols_escapable+backtick_code_blocks+fenced_code_blocks+space_in_atx_header+intraword_underscores+lists_without_preceding_blankline+shortcut_reference_links+yaml_metadata_block
      writer: docusaurus_writer.lua
      output-ext: md
      inline-includes: true
```

---

This doesn't work because of a variety of reasons. JJ and I spent 40 minutes pairing over this. It's complicated :)

---

Just wanted to be sure it was not forgotten ! All good then.